zimui/public/.gitignore

```diff
@@ -0,0 +1,4 @@
+channel.json
```

Comment:
They are not on the fs anymore, this was used only during tests. I removed them from gitignore (but kept the file so that it act as a placeholder for keeping the directory in git, since it is a useful directory in Vue.JS/Vite + it could be helpful for tests / further developments)